### PR TITLE
feat-text-utility

### PR DESCRIPTION
**Pull request by iamsaurav28**
utility

### CODE REVIEW

File: Stylesheets/components/utility.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  font-size: 1.5rem;
+  line-height: 2.5rem;
+}
+.body-cp-md {
+  font-size: 1rem;
+  line-height: 2.5rem;
+}
+.body-cp-sm {
+  font-size: 0.75rem;
+  line-height: 2.5rem;
```
Comment (ttahm3d) [inline Stylesheets/components/utility.css]: If the line-height is same for all the properties, move it to a common property and use it across.

**Review by ttahm3d — COMMENTED** [review]
Have added one comment. Please look into it.

File: Stylesheets/components/utility.css (hunk 1 above)
Comment (rahulcs754) [inline Stylesheets/components/utility.css]: body-cp-md class is not sound good 
or you can set font-l(font lasge) , font-md(font-meditext) like this so you can understand these class by name

Comment (iamsaurav28) [reply Stylesheets/components/utility.css]: oh ya i changed it ty :)

**Review by rahulcs754 — COMMENTED** [review]
Just small change and try to create a name you will understand easly

**Review by ttahm3d — APPROVED** [review]
Looks good now. Can be merged.

File: Stylesheets/components/utility.css (hunk 1 above)
Comment (iamsaurav28) [reply Stylesheets/components/utility.css]: yes u r right but i think  we can use shortform , it is easy to name and maintain.

Comment (rahulcs754) [reply Stylesheets/components/utility.css]: text-lg
text-md
text-sm
these are small and shortform

Comment (iamsaurav28) [reply Stylesheets/components/utility.css]: okay done

**Review by rahulcs754 — APPROVED** [review]
now it looks good. try to always set your name understandable or small

**Pull request merged**